src/nextlabs_sdk/_cloudaz/_component_search.py

```
@@ -1,31 +1,52 @@
+    "/console/api/v1/component/search/listNames/{group}",
+)
+_LIST_NAMES_BY_TYPE_SPEC = query_paginated(
+    ComponentNameEntry,
+    "/console/api/v1/component/search/listNames/{group}/{component_type}",
+)
+
+_GROUP_ARG = "group"
+_COMPONENT_TYPE_ARG = "component_type"
 
 
 class ComponentSearchService:  # noqa: WPS214
```

Ruff (0.15.21)

[warning] Invalid rule code in `# noqa`: WPS214

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)